docs/release_notes/ibexa_dxp_v4.3.md

```diff
@@ -159,3 +159,13 @@ product-related responses.
 #### Ability to retrieve a customer group
 
 You can now retrieve customer group by implementing the `Ibexa\Contracts\ProductCatalog\CustomerGroupResolverInterface` interface and tagging it with `ibexa.product_catalog.customer_group.resolver`.
+
+## v4.3.5
+
+- The `UserService::updateUserPassword` method now throws `ContentFieldValidationException`
+with array format accessible via `ContentFieldValidationException::getFieldErrors`.
+The format of the returned array is:
```

Comment:
I would actually rephrase it to emphasize that the format has changed. Sounds too much like completely new exception.

_Background: The method always threw that exception on error, but the array structure/shape was incorrect._